cao_van_thao/lesson3/currency-exchange.php

```diff
@@ -0,0 +1,33 @@
+<?php
+    class change{
+        public $InputMoney;
+        public function __construct($InputMoney){
+            $this->InputMoney = $InputMoney;
+        }
+    }
+    class vnd extends change{
+        public function VndToUsd(){

```

Comment:
```suggestion
        public function vndToUsd() {
```
chữ cái đầu tiên ở tên function viết thường, sau dấu `)` có 1 dấu cách